lib/config.js

```diff
@@ -19,8 +29,8 @@ const convertDotStringToObject = (o, s) => {
 
 const config = {
   get(entry) {
-    return convertDotStringToObject(configFile, entry);
-  }
+    return convertDotStringToObject(getParsedConfig(), entry);
+  },
 };
```

Comment:
Hmmm, can you elaborate on this change a little? Isn't config read just once? (and even if not, won't the `parsedConfig` object stick around to avoid the second readFile?)

In any case, I would drop in our beautiful `memoize` function to abstract away the custom memoization in the function 😉

Reply:
So we can't define `configFile` anymore, since that's imported and after the import the `cwd` changes. So that's why I'm fetching it 'at least after that `cwd` is properly set.

Reply:
Hmm, but if it's read once, why the memoization? Might be fine to just KISS?

Reply:
> Hmm, but if it's read once, why the memoization? Might be fine to just KISS?

Hmm, the config is 'parsed' once; might be a better way of stating it 👍 
All tasks then convert their dot-string-notated (`config.get('tasks.sass.autoprefixer')`) entries and query that object, so it's 'read' lots of times.

Reply:
Say no more! Makes a lot of sense 👍